src/app/features/test-records/amend/views/test-type-select-wrapper/test-type-select-wrapper.component.spec.ts

```diff
@@ -16,7 +19,7 @@ describe('TestTypeSelectWrapperComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       declarations: [TestTypeSelectWrapperComponent, TestTypeSelectComponent],
-      imports: [RouterTestingModule],
+      imports: [RouterTestingModule, HttpClientTestingModule, StoreModule.forRoot({ technicalRecord: vehicleTechRecordReducer })],
       providers: [{ provide: TestTypesService, useValue: { selectAllTestTypes$: of([]), testTypeIdChanged: () => {} } }],
```

Comment:
```suggestion
      imports: [RouterTestingModule, HttpClientTestingModule],
      providers: [
          provideMockStore<State>({ initialState: initialAppState }),
          { provide: TestTypesService, useValue: { selectAllTestTypes$: of([]), testTypeIdChanged: () => {} } }
      ],
```
Generally better practice to mock dependencies for unit tests as you only want to test your specific changes in isolation - errors in these may make it seem like your code is broken, when it isn't.

Reply:
Yeah, you are right. I am not pretty much sure about the unit test. Thank you so much for pointing it out to me. I have amended it now.